pt/news/_posts/2016-11-09-ruby-2-4-0-preview3-released.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: news_post
+title: "Lançado o Ruby 2.4.0-preview3"
+author: "naruse"
+translator: "Filipe Rocha"
+date: 2016-11-09 18:00:00 +0900
+lang: pt
+---
+
+Temos o gosto de anunciar a versão 2.4.0-preview3 do Ruby.
+
+A versão 2.4.0-preview3 do Ruby é a terceira *preview* do Ruby 2.4.0,
+lançada com o fim de obter *feedback* da comunidade.
+
+Poderão
+[enviar *feedback*](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)
+uma vez que ainda podem influenciar das funcionalidades.
+
+## [Introdução de melhorias nas hash tables por Vladimir Makarov](https://bugs.ruby-lang.org/issues/12142)
+
+Melhorar a estrutura interna da hash table (st_table) introduzindo o *open addressing*
+e introdução de um *array* com a ordem de inclusão.
```

Comment:
I think it's missing one sentence here.

`This improvement has been discussed with many people, especially with Yura Sokolov.`